packages/edit-widgets/src/store/selectors.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { toArray } from 'lodash';
+
+export function getWidgetAreas( state ) {
+	return toArray( state.widgetAreas );
+}
+
+export function getWidgetAreaBlocks( state, widgetAreaId ) {
```

Comment:
I wonder if this would be clearer if named `getBlocksPerWidgetArea` or something like that?